src/view.c

```diff
@@ -797,7 +809,7 @@ view_apply_natural_geometry(struct view *view)
 	assert(view_is_floating(view));
 
 	struct wlr_box geometry = view->natural_geometry;
-	view_adjust_floating_geometry(view, &geometry);
+	adjust_floating_geometry(view, &geometry, false);
```

Comment:
Inline comments for `false`, same for the next 2 `adjust_floating_geometry()` calls.